content/docs/_index.md

```diff
@@ -64,19 +64,19 @@ CSM is made up of multiple components including modules (enterprise capabilities
 
 ## CSM Supported Modules and Dell CSI Drivers
 
-| Modules/Drivers                                                                              | CSM 1.5 | [CSM 1.4](../v1/) | [CSM 1.3.1](../v2/) | [CSM 1.2.1](../v3/) |
+| Modules/Drivers                                                                              | CSM 1.6 | [CSM 1.5.1](../v1/) | [CSM 1.4](../v2/) | [CSM 1.3](../v3/) |
 | -------------------------------------------------------------------------------------------- | :-----: | :---------------: | :---------------: | :-----------------: |
 | [Authorization](https://hub.docker.com/r/dellemc/csm-authorization-sidecar)                  | v1.5.0  |      v1.4.0       |      v1.3.0       |       v1.2.0        |
```

Comment:
Hi @shanmydell 
Please correct the versions of CSM modules for previous releases. They need to be updated accordingly when a new release/column is added.